backend/src/main/java/corea/matching/repository/MatchResultRepository.java

```diff
@@ -2,12 +2,20 @@
 
 import corea.matching.domain.MatchResult;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
 
 import java.util.List;
 import java.util.Optional;
 
 public interface MatchResultRepository extends JpaRepository<MatchResult, Long> {
 
+    @Query("""
+            SELECT COUNT(m) > 0
+            FROM MatchResult m
+            WHERE m.roomId = :roomId AND (m.reviewer.id = :memberId OR m.reviewee.id = :memberId)
+            """)
+    boolean existsByRoomIdAndMemberId(long roomId, long memberId);
```

Comment:
현재 작성된 쿼리는 모든 MatchResult를 순회하며 마지막에 결과를 반환하게 됩니다.

EXISTS 문을 사용하는건 어떨까요?? 😀
```suggestion
@Query("""
    SELECT EXISTS(
        SELECT 1
        FROM MatchResult m
        WHERE m.roomId = :roomId AND (m.reviewer.id = :memberId OR m.reviewee.id = :memberId)
    )
""")
    boolean existsByRoomIdAndMemberId(long roomId, long memberId);
```